src/main.c

```diff
@@ -16,16 +16,18 @@
 #include "common.h"
 #include "protocol/interface.h"
 #include "mixer.h"
-#include "gui/gui.h"
-#include "buttons.h"
+#include "gui/gui.h" //diff1
+#include "buttons.h" //diff2
+                     //diff3
 #include "timer.h"
-#include "autodimmer.h"
-#include "music.h"
+#include "autodimmer.h" //diff4
+#include "music.h"      //diff5
```

Comment:
Lint Error:
main.c:19:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:19:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:20:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:20:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:21:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:23:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:23:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:24:  Should have a space between // and comment  [whitespace/comments] [4]